package.json

```diff
@@ -37,7 +43,8 @@
     "gulp-coffee": "^2.2.0",
     "gulp-coffeelint": "^0.6.0",
     "gulp-shell": "^0.5.2",
-    "require-npm4-to-publish": "^1.0.0"
+    "require-npm4-to-publish": "^1.0.0",
+    "typescript": "^2.5.3"
```

Comment:
Is there a reason we didn't got for ^2.6 ?

Reply:
This is what was in my npm5 cache, and I installed it whilst flying over Sardinia on Thursday :smiley:. Good reminder - I've updated it to the latest.

Reply:
Wow, so npm5's offline cache works! That's awesome!